src/blueapi/config.py

```diff
@@ -63,6 +71,7 @@ class EnvironmentConfig(BlueapiBaseModel):
         Source(kind=SourceKind.PLAN_FUNCTIONS, module="dodal.plan_stubs.wrapped"),
     ]
     events: WorkerEventConfig = Field(default_factory=WorkerEventConfig)
+    metadata: MetadataConfig = Field(default_factory=MetadataConfig)
```

Comment:
We should think carefully about the defaults here, don't want people getting documents that say their data comes from beamline "p01" just because they didn't know there was an option to change it. Might be worth a quick discussion.

Reply:
Default to `None` instead.